### ISSUE TITLE
Aplication does not update apps that were dragged to extensions page.

### ISSUE THREAD

**Issue by Giger22** [NeverDecaf/chromium-web-store#116]
Application does not update apps that was not installed through the Chromium Web Store. (i.e. if you drag to extensions page).
For example:
https://github.com/iamadamdev/bypass-paywalls-chrome
but this also happens if apps is available to install through the chrome Web Store.

**Comment by NeverDecaf** [comment]
Does this ONLY affect extensions installed from outside the web store? or does it also affect extensions from the web store? If the former then it may be a permissions issue, IIRC you should be able to click a wrench icon when the installation fails to request additional permissions. If it is the latter, you need to set the `#extension-mime-request-handling` flag.

**Comment by Giger22** [comment]
This affect BOTH extensions (from the web store and outside). It is not a problem like I get an error. The extension is in the Chromium web store list but it never even gets a prompt to get updated.
I can give an example when this happens. The extension does not even try to update when in the "details" page of an extension there is something like this (when you drag .crx file or .zip):
**Source**
Unpacked extension
Loaded from: ~/Downloads/Chromium extensions/bypass-paywalls-chrome-master

and when app is installed directly through chromium web store, there is
**Source**
Not from Chrome Web Store.

In other words:
For apps from chrome web store you can fix this by removing app that was dragged and install it through chromium web store, although it is mundane task if you have a lot of extensions. 
For some time I though it was a serious bug so I think it should be fixed in some way.
I think it would be feature request to port dragged extensions to chromium web store, so it can update them.

For apps that can't be installed with the new button "Add to Chromium" I do not see a fix even that extension in listed in chromium web store.

**Comment by NeverDecaf** [comment]
I tried installing an older version of the extension you linked using both methods (drag+drop and "load unpacked") and in both cases chromium web store correctly shows the available update.
![image](https://user-images.githubusercontent.com/19150416/218838987-e80881e8-5b1c-4169-8d1d-c4b96245459f.png)
![image](https://user-images.githubusercontent.com/19150416/218838717-ecfbb9ed-86c9-48c7-a7af-ae95a33326c9.png)
If the issue is only affecting this extension in particular (or only some extensions), it is because the author has not updated the version number in [updates.xml](https://github.com/iamadamdev/bypass-paywalls-chrome/blob/master/src/updates/updates.xml) since version 1.7.9. Can you try installing an earlier version of Bypass Paywalls (1.7.8 or earlier) and check if the issue still occurs?

**Comment by NeverDecaf** [comment]
I should add that the majority of extensions from outside the web store do not provide an `updates.xml` meaning there is simply no way to check for updates to those extensions. You can verify this yourself by checking an extension's `manifest.json` for the `update_url` field.

**Comment by Giger22** [comment]
> If the issue is only affecting this extension in particular (or only some extensions), it is because the author has not updated the version number in [updates.xml](https://github.com/iamadamdev/bypass-paywalls-chrome/blob/master/src/updates/updates.xml) since version 1.7.9. Can you try installing an earlier version of Bypass Paywalls (1.7.8 or earlier) and check if the issue still occurs?

Yeah, I checked earlier version and issue does not occur. Thanks!

**Comment by Giger22** [comment]
Ok, I thought I have ufw so running this command:
firewall-cmd --zone=trusted --add-interface=waydroid0
fixed issue.
